core/3-binary-data/project.lock.json

```diff
@@ -170,12 +170,12 @@
           "Zlib.Portable.Signed": "1.11.0"
         },
         "compile": {
-          "lib/net45/Google.Apis.dll": {},

```

Comment:
I dunno.  I didn't intentionally update anything.  I just added the project to the solution, and rebuilt.